src/components/ExecuteButton.scss

```diff
@@ -0,0 +1,7 @@
+.ins-c-remediations-connection-status {
+  margin-right: 5px;
```

Comment:
Use `var(--pf-global--spacer--xs)` instead of direct number, also align this to one line.